### ISSUE TITLE
Context.Consumer not currently accessible in nextjs project

### ISSUE THREAD

**Issue by taysea** [hpe-design/aries#34]
TLDR: Using <ResponsiveContext.Consumer> to create responsive layout works, but using the useContext hook does not.

Unable to get size of window because Nextjs uses SSR which doesn't have knowledge of window width. Because of this, adding a useContext to control responsive behavior doesn't seem to be having an effect. I think with some added configuration, we will get it working but I need to find a resource that explains how.

There seems to be some discussion around this:
- https://reacttricks.com/sharing-global-data-in-next-with-custom-app-and-usecontext-hook/
- https://github.com/zeit/next.js/issues/7167
- https://github.com/zeit/next.js/issues/4194
